### PR TITLE
Add light documentation on running a manual scale test

### PR DESCRIPTION
**Pull request by dvail**
## Description

This consolidates some notes I have from past experience scale testing for UI features, and adds a short README to the `scale` directory. This also contains a lightly commented `workload.yaml` that gives a bit of information on what each property does.

## Testing Performed
NA

### CODE REVIEW

**Comment by dvail** [comment]
Current dependencies on/for this PR:
* master
  * **PR #5200** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/5200" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/5200?utm_source=stack-comment).

File: scale/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+To start a scale simulation, run the following from the root of the stackrox repo:
+
+```sh
+cd scale
+./launch_workload.sh <workload_name>
+# <workload_name> is the name of a yaml file in the `workloads` directory, without file extension
+# e.g. $ ./launch_workload.sh xlarge
+```
+
+**Note that this is a destructive operation on your current active `kubectx` cluster that cannot be easily undone.**
```
Comment (dashrews78) [inline scale/README.md:14]: I'd probably put the Note above the command

**Review by dashrews78 — APPROVED** [review]
Left one comment, mainly because I would read up to the command, execute it, then continue reading.

File: scale/README.md (hunk 1 above)
Comment (dvail) [reply scale/README.md]: Good call 👍

**Review by connorgorman — APPROVED** [review]

**Pull request merged**